src/content/en/tools/chrome-devtools/console/command-line-reference.md

```diff
@@ -80,6 +81,14 @@ property:
 
 ![Example of $('img').src](images/selector-img-src.png)
 
+This function also supports a second parameter, startNode, that specifies an element or Node from which to search for elements. The default value of this parameter is `document`. 
```

Comment:
Please wrap lines at no more than 80 characters

Reply:
@petele I think the bot allows up to 100 characters per line...

![image](https://user-images.githubusercontent.com/4713486/42179758-cd5fdab0-7de9-11e8-84c9-3d3442888d6d.png)

Reply:
It does allow up to 100 so that there's some flexibility, but 80 is the preferred line length.